code/datums/crew_objective.dm

```diff
@@ -307,11 +330,143 @@ ABSTRACT_TYPE(/datum/objective/crew/janitor)
 					return 0
 			return 1
 
-//	bartender
+#define DRINK_OBJ_DONE 7
+#define DRINK_OBJ_1 1
+#define DRINK_OBJ_2 2
+#define DRINK_OBJ_3 4
```

Comment:
Define the number of drinks as well, you reference `3` in a bunch of places for `/datum/objective/crew/bartender/drinks`

Reply:
Done!